tests/cases/stable/runners/register_no_perm.py

```
@@ -0,0 +1,10 @@
+# { "Depends": "py-genlayer:test" }
+from genlayer.vm import register_runner
+
+# default permissions (rwscn) do NOT grant `register_runners` (u)
+try:
+	register_runner(b'# { "Depends": "py-genlayer:test" }\nx\n')
+	print('registered')
+except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)